controller/src/controller/config.py

```diff
@@ -498,6 +499,123 @@ class EnabledFileserverConfig(FileserverConfig):
     ] = []
 
 
+class FSAdminConfig(BaseModel):
+    """Configuration for filesystem administration environment."""
+
+    model_config = ConfigDict(alias_generator=to_camel, populate_by_name=True)
+
+    affinity: Annotated[
+        Affinity | None,
+        Field(
+            title="Affinity rules",
+            description="Node and pod affinity rules for file server pods",
+        ),
+    ] = None
+
+    application: Annotated[
+        str | None,
+        Field(
+            title="Argo CD application",
+            description="Argo CD application into which to put fsadmin pods",
+        ),
+    ] = None
+
+    extra_annotations: Annotated[
+        dict[str, str],
+        Field(
+            title="Extra annotations",
+            description=(
+                "Extra annotations to add to all user file server ``Job`` and"
+                " ``Pod`` Kubernetes resources"
+            ),
+        ),
+    ] = {}
+
+    # We default this so we don't need a configuration-file flag day synced
+    # to the nublado updates.
+    image: Annotated[
+        ContainerImage,
+        Field(
+            title="fsadmin Docker image",
+            description="Docker image to run as fsadmin",
+        ),
+    ] = ContainerImage(
+        repository="ghcr.io/lsst-sqre/nublado-fsadmin",
+        pull_policy=PullPolicy.ALWAYS,
+        tag="latest",
+    )
+
+    mount_prefix: Annotated[
+        str | None,
+        Field(
+            title="Prefix for fsadmin mounts",
+            description=(
+                "If given, the mounts will be collected under this directory."
+            ),
+            examples=["/mnt"],
+        ),
+    ] = None
+
+    node_selector: Annotated[
+        dict[str, str],
+        Field(
+            title="File server node selector",
+            description=(
+                "Labels that must be present on Kubernetes nodes for any"
+                " fsadmin pods to be scheduled there"
+            ),
+            examples=[{"disktype": "ssd"}],
+        ),
+    ] = {}
+
+    pod_name: Annotated[
+        str,
+        Field(
+            title="fsadmin pod name",
+            description="Pod/container name for fsadmin",
+        ),
+    ] = "fsadmin"
```

Comment:
`nublado-fsadmin` maybe?

Reply:
We already know it's in the nublado namespace.  Saves eight characters of typing when you want to exec into the pod.